### ISSUE TITLE
Vehicle inspection 

### ISSUE THREAD

**Issue by Stretch379** [TeleportXYZ/TRIP-Guides#48]
It has come to my attention by using both Uber and Lyft platforms. The vehicle inspections should be done routinely, and after every incident whether reported to the insurance or not to make sure that the vehicle is safe, and eye appealing for the customers ### What's missing from the TRIP Guides

**Comment by hwhelchel** [comment]
@Stretch379 Gary so it sounds like your proposal is for there to be a section in the Guides on vehicle inspection policy for drivers and/or operators.

@ivandiku I imagine vehicle inspection policies are regulations that TNC, TNP, or Fleet Managers need to comply with. So this might vary from state to state? Is that correct?

**Comment by Stretch379** [comment]
As far as it being mechanically sound, and not a complete eyesore, yes

**Comment by Alejandro1954-ops** [comment]
> @Stretch379 Gary so it sounds like your proposal is for there to be a section in the Guides on vehicle inspection policy for drivers and/or operators.
> 
> @ivandiku I imagine vehicle inspection policies are regulations that TNC, TNP, or Fleet Managers need to comply with. So this might vary from state to state? Is that correct?

@hwhelchel I will let @ivandiku chime in but 100% correct to your second question -compliance is on the TNC, TNP, or Fleet Managers and it does vary from state to state.

**Comment by Stretch379** [comment]
Yes, there are variances from state to state however, some of these riders are not properly reporting damage vehicles after the ride is over or while they are in the ride and just letting people get by with it. My proposal here is to try and keep an honest open network of drivers that want to maintain and take care of their vehicles.Sent from my iPhoneOn Feb 17, 2024, at 4:42 PM, AlejandroCamacho ***@***.***> wrote:﻿

@Stretch379 Gary so it sounds like your proposal is for there to be a section in the Guides on vehicle inspection policy for drivers and/or operators.
@ivandiku I imagine vehicle inspection policies are regulations that TNC, TNP, or Fleet Managers need to comply with. So this might vary from state to state? Is that correct?

@hwhelchel I will let @ivandiku chime in but 100% correct to your second question -compliance is on the TNC, TNP, or Fleet Managers and it does vary from state to state.

**Comment by Alejandro1954-ops** [comment]
@Stretch379 fair point. I am not sure how TRIP can enforce this policy. Yearly inspections? Regardless, I can attest from personal experience that Uber was not thorough in routine inspections both as a passenger and a driver. It was a pain just to connect with someone on their team to get information about it.

**Comment by Stretch379** [comment]
I would say every 18 months mandatory inspection as well as inspection after every safety concern or incident/accident  reported. I know I have gotten a safety flag on Lyft and due to their process not being able to know what it was. If I knew what the safety flag was, I could get it fixed properly however, knowing what I do about vehicles, there is nothing safety related there was wrong it was probably just an upset customer.  Sent from my iPhoneOn Feb 18, 2024, at 3:23 PM, AlejandroCamacho ***@***.***> wrote:﻿
@Stretch379 fair point. I am not sure how TRIP can enforce this policy. Yearly inspections? Regardless, I can attest from personal experience that Uber was not thorough in routine inspections both as a passenger and a driver. It was a pain just to connect with someone on their team to get information about it.

**Comment by Alejandro1954-ops** [comment]
The 18 months is very reasonable. I think the only barrier is that TRIP will need to partner with a company like JiffyLube to make sure these inspections are professionally done. It was the same with Uber.

**Comment by Stretch379** [comment]
In all honesty, Jiffy Lube, take five and quick car are all bad places that don’t go by the book just speaking from experience I’ve been a mechanic since I was 16 and wouldn’t take any vehicle to any of those placesSent from my iPhoneOn Feb 19, 2024, at 8:55 AM, AlejandroCamacho ***@***.***> wrote:﻿
The 18 months is very reasonable. I think the only barrier is that TRIP will need to partner with a company like JiffyLube to make sure these inspections are professionally done. It was the same with Uber.

**Comment by Alejandro1954-ops** [comment]
Haha, fair enough. And I believe you are 100% correct. But I just don't know how TRIP will be enable to realistically enforce this if they don't partner with a big auto repair shop company.

**Comment by hwhelchel** [comment]
@ivandiku when you have a moment can you get up to speed on this discussion and bring some clarity to it with a comment sharing your expertise?
